src/xdocs/config_coding.xml

```diff
@@ -3623,6 +3623,216 @@ class TestHashCode {
       </subsection>
     </section>
 
+    <section name="MatchXpath">
+      <p>Since Checkstyle 8.39</p>
+      <subsection name="Description" id="MatchXpath_Description">
+        <p>
+          Evaluates Xpath query and report violation on all matching AST nodes. This check allows
+          user to implement custom checks using Xpath. If Xpath query is not specified explicitly,
+          then the check does nothing.
+        </p>
+        <p>
+          It is better to define custom messages for violations to explain what is not allowed and
```

Comment:
`It is recommended to define custom message for violation ....`

Reply:
done 👍